server/api/core/email/email.js

```diff
@@ -27,27 +26,10 @@ export function send(options) {
  * @returns {Object} returns source
  */
 export function getTemplate(template) {
-  check(template, String);
```

Comment:
that's a pretty big assumption. it wasn't being used in the default case -> but this was the recommended way for overriding the default email templates up until this point -> and also see: https://github.com/reactioncommerce/reaction/pull/1362